src/commands/github/actionCommands.ts

```diff
@@ -3,24 +3,23 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { Octokit } from "@octokit/rest";
 import { IActionContext } from "vscode-azureextensionui";
 import { ext } from "../../extensionVariables";
 import { ActionTreeItem } from "../../tree/ActionTreeItem";
-import { createGitHubRequestOptions, getGitHubAccessToken, gitHubWebResource } from "../../utils/gitHubUtils";
 import { localize } from "../../utils/localize";
-import { requestUtils } from "../../utils/requestUtils";
+import { createOctokitClient } from "./createOctokitClient";
 
 export async function rerunAction(context: IActionContext, node?: ActionTreeItem): Promise<void> {
     if (!node) {
         node = await ext.tree.showTreeItemPicker<ActionTreeItem>(ActionTreeItem.contextValue, { ...context, suppressCreatePick: true });
     }
 
-    const token: string = await getGitHubAccessToken();
-    const gitHubRequest: gitHubWebResource = await createGitHubRequestOptions(token, node.data.rerun_url, 'POST');
     const rerunRunning: string = localize('rerunRunning', 'Rerun for action "{0}" has started.', node.data.id);
     ext.outputChannel.appendLog(rerunRunning);
 
-    await requestUtils.sendRequest(gitHubRequest);
+    const client: Octokit = await createOctokitClient();
+    await client.actions.reRunWorkflow({ owner: node.data.repository.owner.login, repo: node.data.repository.name, run_id: node.data.id });
```

Comment:
Just double-checking - does this `owner.login` thing work for both user and org repositories?

Reply:
Yep yep, owner.login will be what you need to pass in for both cases.  This is an org:

![image](https://user-images.githubusercontent.com/5290572/94867780-bba62c80-03f6-11eb-823d-a60a871738ea.png)